### PR TITLE
Add Tchibo coffee shop

### PR DESCRIPTION
**Pull request by PiotrBlachnio**

### CODE REVIEW

File: brands/amenity/cafe.json [hunk 1]
```diff
@@ -1062,6 +1062,29 @@
       "takeaway": "yes"
     }
   },
+  "amenity/cafe|Tchibo": {
+    "locationSet": {"include": [
+      "at",
+      "cz",
+      "hu",
+      "is",
+      "nl",
```
Comment (RobinLinde) [inline brands/amenity/cafe.json:1071]: According to the [Dutch Wikipedia article](https://nl.wikipedia.org/wiki/Tchibo#Geschiedenis), the last location in the Netherlands closed in 2007.

Comment (PiotrBlachnio) [reply brands/amenity/cafe.json]: Updated.

**Comment by UKChris-osm** [comment]
Tchibo appears to already exists within the brand index, is this intended to be an update?

**Comment by bhousel** [comment]
> Tchibo appears to already exists within the brand index, is this intended to be an update?
> 
> Edit: I see now, it's already listed but under "_shop/cafe_" as opposed to "_amenity/cafe_"

I'll just merge as-is and cleanup in a followup commit after the identifiers settle..

**Comment by bhousel** [comment]
whoops - I meant to merge but clicked the wrong button.
Anyway I will move the record to `amenity/cafe` later.

**Pull request closed without merging**